examples/README.md

```diff
@@ -44,3 +44,17 @@ copy the `/examples/algob.confg-template.js` to `/examples/algob.config-local.js
 the following positions in the latter file:
 
 + `master-account`: must be an account with some ALGOs - it will be used for deployment and for activating / funding other accounts.
+
+### Examples Description
+
+- [asa](./asa) - This `javascript` project shows how to create Algorand Standard Asset (ASA).
+- [crowdfunding](./crowdfunding) - This `javascript` project demonstrates how to create a Crowdfunding Stateful Smart Contract Application. It's based on a [tutorial](https://developer.algorand.org/solutions/
+- [multisig](./multisig) - This `javascript` example demonstrates authorizing transactions based on logic signature signed by multiple accounts.
+- [nft](./nft) - Non-Fungible-Token Example using stateful TEAL in `javascript`. In this example, we create a new non-fungible-token represented by a name and a ref.
+- [permissioned-voting](./permissioned-voting) -  This `javascript` project demonstrates how to create a Permissioned Voting Stateful Smart Contract Application.
+Original tutorial can be found [here](https://developer.algorand.org/solutions/example-permissioned-voting-stateful-smart-contract-application/)
+- [ref-templates](./ref-templates) - The goal of this project is to present how to work with ASC in `algob` using the best practices and templates using the Algorand reference templates.This is a `javascript` project.
```

Comment:
I think except `htlc` all projects are `js` projects. Can we list them in separate subsections? 
(`# Javascript [projects..], # Typescript [projects..]`)

Reply:
i don't have preference. can separate as well ?

Reply:
We are repeatedly using `this is javascript project` in these points, so i thought having separate section could be better. I don't have a strong preference, we can keep this as well.

Reply:
How about using a tag? I will be more readable. We can do it like this:

- [JS] [permissioned-voting](./permissioned-voting) - project that demonstrates how to create a Permissioned Voting Stateful Smart Contract Application.
- [JS] [link](link) - other project description
- [TS] [link](link) - other project description

Reply:
yes